committee-steering/sig-governance.md

```diff
@@ -0,0 +1,465 @@
+
+When possible, it is preferred that the Admin group represent a
+diverse set of organizations.  This is predicated on interest in
+the role from multiple organizations with qualified candidates.
+
+### Technical Lead role
+
+Canonical list of admins: kubernetes-sig-~~foo~~-techleads@googlegroups.com
+
+The SIG ~~Foo~~ technical leads group is responsible for resolving
+technical disputes and setting technical direction for the SIG.
+Technical leads should be well versed in most technical areas
+governed by the SIG and have shown consistent excellent technical
+judgement and leadership over a sustained period.  Technical leads
+should be focused on directing others within the SIG and are
+ultimately responsible for the technical direction.
+
+Technical leads may appear in OWNERs files as approvers, but not
+every member who is an approver is a technical lead for the SIG.
+
+Technical leads may also be Administrators, however the membership
+and responsibilities of each role is tracked separately.
+
+Technical leads that have held the position for a year and step down may
+retain the title "SIG ~~Foo~~ administrator emeritus".
+
+#### Internally facing responsibilities
+
+- Act as role model within the SIG
+  - Other members will look up to technical leads in the SIG.  Leads
+    must model respectful behavior towards all members of the community
+- Act as an escalation point for technical disputes within the SIG
+- Ensure technical excellence within area
+  - Actively working to reduce complexity of the area
+  - Defining general acceptance criteria for work
+    - code style guide
+    - go doc style guide
+    - test requirements
+    - documentation requirements
+  - Make technical decisions either with other technical leads
+  - Make technical decisions unilaterally if critical and urgent need arises
+  - Help review and approve PRs + resolve discussion on PRs
+  - Provide technical guidance and insight to discussion taking place on GitHub
+- Drive product excellence within an area
+  - Ranking priorities for the SIG
+  - Defining quarterly / yearly goals
+  - Understanding out artifacts published by the SIG are consumed and
+    how to improve that experience
+- Participate in decision making - either voicing support or
+  opposition of proposals
+  - Attend SIG meetings - attendance of every other SIG meeting is expected
+  - Be responsive to messages sent to SIG mailing list
+- Onboard new technical leads and SIG members
+  - Delegate responsibility and ensure opportunities for members to grow
+
+#### Relationship
+
+Technical leads should meet or communicate regularly as a group to
+ensure they share the same priorities and direction.  Decisions made
+by technical leads should be consistent across the group.  If
+technical leads are providing inconsistent direction, then they must
+meet frequently and align themselves.
+
+#### Selecting technical leads
+
+SIG ~~Foo~~ maintains a group of 3-7 technical leads.
```

Comment:
Can we change this to `2-7` or are we trying to go for an odd number? I only ask because some SIGs are very small and may have a hard time having tech leads who can meet all of the SIG attendance requirements.

Reply:
Sure, I meant for SIGs to change this number as needed, but happy to change the default.